components/Styled-Components/ChildNext.js

```diff
@@ -1,12 +1,10 @@
 import styled from "styled-components";
 
 const ChildNext = styled.button`
-  display: flex;
-  text-align: center;
-  justify-content: center;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 10rem;
+  position: absolute;
```

Comment:
Great work on the positioning/styling for the child next button, which then keeps it into the same place for each page ✨